### PR TITLE
adds rsyslog files

### PR DESCRIPTION
**Pull request by angrieralien**
* Adds in files required to build rsyslog image 
* Rsyslog image is configured to support the named pipe abstraction in `audito-maldtio`

### CODE REVIEW

**Review by JAORMX — CHANGES_REQUESTED** [review]
This LGTM overall. I'd suggest moving this to a `contrib/` directory as other projects do to indicate that these reference files belong to an external system.

File: rsyslog/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:22.04
```
Comment (JAORMX) [inline rsyslog/Dockerfile:1]: Is Ubuntu needed here? Is there an rsyslog package in alpine?

**Review by JAORMX — CHANGES_REQUESTED** [review]

File: rsyslog/Dockerfile (hunk 1 above)
Comment (angrieralien) [reply rsyslog/Dockerfile:1]: seems to be an rsyslog pkg in alpine:3.17.3

Comment (angrieralien) [reply rsyslog/Dockerfile:1]: we dont need ubuntu. i do like ubuntu :)

**Review by JAORMX — APPROVED** [review]

**Pull request merged**